go/engine/pgp_import_key_test.go

```diff
@@ -188,6 +188,40 @@ func TestPGPImportPrivImport(t *testing.T) {
 	}
 }
 
+func TestPGPImportLocalPrivateThenServer(t *testing.T) {
+	tc := SetupEngineTest(t, "pgpsave")
+	defer tc.Cleanup()
+
+	user := CreateAndSignupFakeUser(tc, "login")
+	secui := &libkb.TestSecretUI{Passphrase: user.Passphrase}
+	uis := libkb.UIs{LogUI: tc.G.UI.GetLogUI(), SecretUI: secui}
+
+	_, _, key := genPGPKeyAndArmor(t, tc, user.Email)
+
+	// Import a private key locally first.
+	eng, err := NewPGPKeyImportEngineFromBytes(tc.G, []byte(key), false /* pushSecret*/)
+	require.NoError(t, err)
+	mctx := NewMetaContextForTest(tc).WithUIs(uis)
+	err = RunEngine2(mctx, eng)
+	require.NoError(t, err)
+
+	kid := eng.GetKID()
+
+	// Try to import with push secret afterwards - user also wants
+	// to have this key available online.
+	eng, err = NewPGPKeyImportEngineFromBytes(tc.G, []byte(key), true /* pushSecret*/)
+	require.NoError(t, err)
+	mctx = NewMetaContextForTest(tc).WithUIs(uis)
+	err = RunEngine2(mctx, eng)
+	require.NoError(t, err)
+
+	ss, err := mctx.ActiveDevice().SyncSecretsForce(mctx)
+	require.NoError(t, err)
+	privKey, ok := ss.FindPrivateKey(kid.String())
+	require.True(t, ok)
+	require.NotEmpty(t, privKey.Bundle)
```

Comment:
i don't know if this is a good check - secret syncer here seems to only use server side keys using `key/fetch_private` but maybe i missed something.